CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## Unpublished
+
+* Don't add Zip64 fields when copying entries from another archive
```

Comment:
Please add a link to this PR, similar to how it is done in other entries of the changelog.